e2e.py

```diff
@@ -0,0 +1,74 @@
+"""Simple Bigtable client demonstrating auto-generated veneer.
+"""
+
+import argparse
+import logging
+
+from google.bigtable.v1 import bigtable_service_api
+from google.bigtable.admin.cluster.v1 import bigtable_cluster_service_api
+from google.bigtable.admin.table.v1 import bigtable_table_service_api
+
+from google.bigtable.v1 import bigtable_data_pb2 as data
+from google.bigtable.admin.cluster.v1 import bigtable_cluster_data_pb2 as cluster_data
+from google.bigtable.admin.table.v1 import bigtable_table_data_pb2 as table_data
+
+def run(project_id):
+  with bigtable_service_api.BigtableServiceApi() as bigtable_api, \
+       bigtable_cluster_service_api.BigtableClusterServiceApi() as cluster_api, \
+       bigtable_table_service_api.BigtableTableServiceApi() as table_api:
```

Comment:
If this is needed routinely for common scenarios, it could be a problem with the API.

If the scenarios are fairly affinitized the with the API, I would change the sample, to do the with statements one a time.